config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-   root 'users#index'
-   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-   end
+  root "users#index"
+  
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create] do
```

Comment:
- [**Optional**] For now we can limit ourselves to the routes required to create a post(`resources :posts, only: [:new, :create] do`) but I think there's no problem since you also added map actions for the others